### DISCUSSION TITLE
More than one 'chat-link'?

### DISCUSSION THREAD

**Discussion by cpengilly** [shuding/nextra#2320] [General]
Hi! I'd like to be able to link to my **twitter** and **discord** channels from the top nav, but it seems that the chat/link attribute can only be used once. Is there a workaround by chance? I figure we're not the only ones who'd like to display icons for git, discord, and twitter in the top-right corner of the theme, so I thought I'd reach out to see what others have done to solve this. Thank you in advance for any assistance.

**Comment by dimaMachina** [comment]
Use `extraContent` https://nextra.site/docs/docs-theme/theme-configuration#customize-the-navbar
